rpm-ostree-container-bootc.sh

```diff
@@ -0,0 +1,106 @@
+#
+# Copyright (C) 2023  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+
+# Ignore unused variable parsed out by tooling scripts as test tags metadata
+# shellcheck disable=SC2034
+TESTTYPE="payload ostree bootc reboot skip-on-rhel-8 skip-on-rhel-10"
+# skip-on-rhel"
+
+. ${KSTESTDIR}/functions.sh
+
+os_major=${KSTEST_OS_VERSION%%.*}
+if [ "${KSTEST_OS_NAME}" == "rhel" ]; then
```

Comment:
@KSTEST_OS_NAME@ and @KSTEST_OS_VERSION@ should be used in .ks.in (https://github.com/rhinstaller/kickstart-tests/blob/master/README.rst#chapter-4-environment-variables)

We updated similar stuff recently and use the variables like this:  https://github.com/rhinstaller/kickstart-tests/blob/208da7737a2662663634855cfa005ae8ccdd8b77/repo-install.ks.in#L20 which is not ideal. I'd maybe add an rhel_os_major_version function to the https://github.com/rhinstaller/kickstart-tests/blob/master/scripts-lib.sh that would replace is_rhel8 and is_rhel9. But that is something  I'd do independently on top of your PR.